Sources/OpenAPIRuntime/Conversion/Converter+Client.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 import Foundation
```

Comment:
Isn't this API breaking? You were previously importing Foundation as `public` but with the changed flag it is now imported as `internal`. Since imports are leaky in Swift this would be API breaking.